### ISSUE TITLE
Problem: decentralization elo rankings are not averaged

### ISSUE THREAD

**Issue by gazhayes** [Blockrazor/blockrazor#223]
Problem: decentralization elo rankings look like they aren't being used, they are not being averaged and copied to the currency collection. Looking in the elorankings collection I see that the decentralization rankings have a null in category, this is probably the cause but I'm not completely sure if that's the only cause. For example item with _id: 94668AADAb.

**Comment by anbud** [comment]
@gazhayes, I can't seem to reproduce this, either locally or on the production server.

For a test, I've just added a new question for decentralization (`Decentralization test`) shown at the codebase page. I've added a few currencies, answered the questions, called `tabulateElo` and `averageElo` for `decentralization` (from modpanel dashboard), and finally called the `updateGraphData` method. Everything works as expected, and the max ELO ranking for decentralization is currently 430. For example, decentralization ranking for ETC is 430, and for XMR is 423 currently.
Also, I couldn't find any ratings with null category.
